modules/backend/behaviors/RelationController.php

```diff
@@ -843,6 +843,9 @@ protected function makeManageWidget()
             }
 
             $widget = $this->makeWidget('Backend\Widgets\Lists', $config);
+            $widget->setSearchOptions([
+                'scope' => $this->getConfig('manage[searchScope]')
+            ]);
```

Comment:
@danielbidala What happens if `searchScope` is not defined in `config_relation.yaml` ?

Reply:
@mjauvin In that case `scope` will be `null` and the default list search will be used based on `getSearchableColumns()` method in `List.php`.